boost-gradle/src/main/groovy/io/openliberty/boost/gradle/tasks/BoostPackageTask.groovy

```diff
@@ -173,21 +175,7 @@ public class BoostPackageTask extends AbstractBoostTask {
         return springBootDependencies
     }
 
-    public String getClassifier() {
```

Comment:
Where did this go? Doesn't something use this method?

Reply:
We weren't using this anywhere so I removed it.